packages/block-directory/package.json

```diff
@@ -21,10 +21,6 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"react-native": "src/index",
-	"sideEffects": [
```

Comment:
This package confuses me. How is it supposed to be used?

Is it meant to be a single side effect, by doing just `import '@wordpress/block-directory';`? I'm seeing a bunch of components in there, but I guess they're all loaded transitively?